src/gpuhunt/providers/cudocompute.py

```diff
@@ -0,0 +1,458 @@
+import asyncio
+from collections import namedtuple
+from concurrent.futures import ThreadPoolExecutor
+from itertools import chain
+from typing import Optional, List
+from cudo_compute import cudo_api
+from gpuhunt import QueryFilter, RawCatalogItem
+from gpuhunt._internal.constraints import KNOWN_GPUS
+from gpuhunt.providers import AbstractProvider
+import logging
+
+CpuMemoryGpu = namedtuple("CpuMemoryGpu", ["cpu", "memory", "gpu"])
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+class CudoComputeProvider(AbstractProvider):
+    NAME = "cudocompute"
+
+    def get(
+            self, query_filter: Optional[QueryFilter] = None, balance_resources: bool = True
+    ) -> List[RawCatalogItem]:
+        offers = asyncio.run(self.fetch_all_vm_types())
+        return sorted(offers, key=lambda i: i.price)
+
+    async def fetch_all_vm_types(self):
```

Comment:
Any reason why to use async instead of just threading?

Reply:
Its my confusion, async is not needed. I checked with just threading and time total taken is equal. I will change it to just threading which much simpler. Thanks